app/search/management/commands/create_search_index.py

```diff
@@ -0,0 +1,30 @@
+'''
+    Copyright (C) 2019 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+from django.core.management.base import BaseCommand
+
+from search.models import SearchResult
+
+
+class Command(BaseCommand):
+
+    help = 'uploads search results into elasticsearch'
+
+    def handle(self, *args, **options):
+        for sr in SearchResult.objects.all():
```

Comment:
One thing thats actually really cool about Django querysets is that it is smart enough to not load all of these objects into memory if you back it with postgres: https://docs.djangoproject.com/en/dev/ref/models/querysets/#iterator

Reply:
good call, so sounds like we should use `.iterator()` here to explicitly do this